superset-frontend/src/views/CRUD/dashboard/DashboardCard.tsx

```diff
@@ -44,7 +44,7 @@ interface DashboardCardProps {
   saveFavoriteStatus: (id: number, isStarred: boolean) => void;
   favoriteStatus: boolean;
   dashboardFilter?: string;
-  userId?: number;
+  userId?: string | number;
```

Comment:
Hey! I don't think that this should ever be a string? Could you give me more context?

Reply:
I found it strange too, but it comes from already existing code: `DashboardList.tsx:73` and I didn't want to dig deep into this for this PR

Reply:
I think this should be number. You can take note of it for a follow up.

Reply:
I think this should be number. You can take note of it for a follow up.